app/uk/gov/hmrc/eoricommoncomponent/frontend/audit/Auditable.scala

```diff
@@ -31,24 +31,6 @@ class Auditable @Inject() (auditConnector: AuditConnector, appConfig: AppConfig)
 
   private val auditSource: String = appConfig.appName
 
-  private val audit: Audit = Audit(auditSource, auditConnector)
-
-  def sendDataEvent(
```

Comment:
I think removing this method means that all of the connectors have now been updated?  I will check with you tomorrow @PatrykRudnicki